### PR TITLE
DISTPG-368-Release-notes-12.9

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   .github/workflows/main.yml
	new file:   docs/release-notes-v12.9.md
	modified:   docs/release-notes.md
	modified:   mkdocs-base.yml

### CODE REVIEW

File: docs/release-notes-v12.9.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+| [pgAudit](https://www.pgaudit.org/)             | 1.4.1   | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[pgAudit set_user](https://github.com/pgaudit/set_user)|3.0.0| provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.36    | a backup and restore solution for PostgreSQL       |
+|[pgBadger](https://github.com/darold/pgbadger)    | 11.6    | a fast PostgreSQL Log Analyzer               |
+| [PgBouncer](https://www.pgbouncer.org/)          | 1.16.1  | a lightweight connection pooler for PostgreSQL      |
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)                                            | 1.0.0 - Beta 2   | collects and aggregates statistics for PostgreSQL and provides histogram information. |
```
Comment (EvgeniyPatlan) [inline docs/release-notes-v12.9.md]: we have RC1

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

File: docs/release-notes-v12.9.md (hunk 1 above)
Comment (nastena1606) [reply docs/release-notes-v12.9.md]: Fixed

File: docs/release-notes-v12.9.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+| [pgAudit](https://www.pgaudit.org/)             | 1.4.1   | provides detailed session or object audit logging via the standard logging facility provided by PostgreSQL                |
+|[pgAudit set_user](https://github.com/pgaudit/set_user)|3.0.0| provides an additional layer of logging and control when unprivileged users must escalate themselves to superuser or object owner roles in order to perform needed maintenance tasks.|
+| [pgBackRest](https://pgbackrest.org/)           | 2.36    | a backup and restore solution for PostgreSQL       |
+|[pgBadger](https://github.com/darold/pgbadger)    | 11.6    | a fast PostgreSQL Log Analyzer               |
+| [PgBouncer](https://www.pgbouncer.org/)          | 1.16.1  | a lightweight connection pooler for PostgreSQL      |
+| [pg_repack](https://github.com/reorg/pg_repack) | 1.4.7   | rebuilds PostgreSQL database objects           |
+| [pg_stat_monitor](https://github.com/percona/pg_stat_monitor)                                            | 1.0.0 - RC   | collects and aggregates statistics for PostgreSQL and provides histogram information. |
```
Comment (EvgeniyPatlan) [inline docs/release-notes-v12.9.md]: it is rc.1 not just rc

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Review by EvgeniyPatlan — APPROVED** [review]

**Review by LenzGr — APPROVED** [review]
Looks good to me - thank you @EvgeniyPatlan for confirming the version numbers!

**Review by hqakhtar — APPROVED** [review]
LGTM

**Pull request merged**